cmd/containerd/builtins_linux.go

```diff
@@ -4,4 +4,5 @@ import (
 	_ "github.com/containerd/containerd/linux"
 	_ "github.com/containerd/containerd/metrics/cgroups"
 	_ "github.com/containerd/containerd/snapshots/overlay"
+	_ "github.com/containerd/cri-containerd"
```

Comment:
I think we should support `no_cri` buildtag for those who wants to reduce the binary size

Reply:
Done